integration-cli/docker_cli_build_test.go

```diff
@@ -770,16 +770,30 @@ RUN [ $(ls -l /exists/exists_file | awk '{print $3":"$4}') = 'dockerio:dockerio'
 }
 
 func TestBuildCopyAddMultipleFiles(t *testing.T) {
-	name := "testcopymultiplefilestofile"
-	defer deleteImages(name)
-	ctx, err := fakeContext(`FROM busybox
+        var url string
+        if os.Getenv("DOCKER_NET") == "none" {
```

Comment:
AFAIK we will try not to use environment variables set in the Dockerfile. #10797 check with @tianon